src/main/java/com/google/devtools/build/lib/packages/Type.java

```diff
@@ -559,6 +559,15 @@ public Map<KeyT, ValueT> convert(Object x, Object what, Object context)
       return ImmutableMap.copyOf(result);
     }
 
+    @Override
+    public Map<KeyT, ValueT> concat(Iterable<Map<KeyT, ValueT>> iterable) {
```

Comment:
Docstring of `Type.concat()` is now misleading - it should be something like `Implementation of concatenation for this type, as if by {@code elements[0] + ... + elements[n-1]} for scalars or lists, or {@code elements[0] | ... | elements[n-1]} for dicts`